### PR TITLE
STI by partner type

### PR DESCRIPTION
**Pull request by kweiss2**

### CODE REVIEW

File: R/mod.sti.R [hunk 1]
```diff
@@ -142,7 +142,16 @@ sti_trans <- function(dat, at) {
...
+  inf.type.gc <- NULL
+  if (sum(trans_ugc, trans_rgc) > 0) {
+      transAL_ugc <- al[allActs_ugc[trans_ugc == 1], drop = FALSE]
+      transAL_rgc <- al[allActs_rgc[trans_rgc == 1], drop = FALSE]
+      all_trans_gc <- rbind(transAL_ugc, transAL_rgc)
+      inf.type.gc <- all_trans_gc[, "ptype"]
+  }
+  
+  
   # Rectal CT -----------------------------------------------------------
```
Comment (smjenness) [inline R/mod.sti.R]: I trust that you tested this out and everything looks right in the output?

**Comment by kweiss2** [comment]
Yes, it's a little different than the script for HIV because I was troubleshooting in the debugger and it looks right. Here's the equivalent HIV code for comparison:


```
inf.type <- NULL
if (sum(trans.ip, trans.rp) > 0) {
    ....
    inf.type <- c(disc.ip[trans.ip == 1, "ptype"],
                  disc.rp[trans.rp == 1, "ptype"])
    ....
```

**Comment by smjenness** [comment]
Ok, but it's generating an error and I don't understand why that isn't showing up on your end during  your testing. Please check again...

**Comment by kweiss2** [comment]
Still having issues on Windows, but on my Mac laptop, I was able to Build and Reload, see the error, and then address it

**Comment by smjenness** [comment]
Regarding the Windows issue, can you try removing the RcppExports files in both the R/ and source/ folders and try rebuilding?

**Comment by smjenness** [comment]
Closing this out without merge for now because we didn't need it for the STI paper, but we can always come back to it to add it back in...

**Pull request closed without merging**